docs/release-notes/RELEASE-NOTES-0.0.0-iris.md

```diff
@@ -0,0 +1,79 @@
+# About this release
+
+[This](https://github.com/aeternity/aeternity/releases/tag/v0.0.0-iris) is a Iris test release.
```

Comment:
Is this tag correct?

Reply:
in what context ? it is semver compatible

Reply:
Just making sure "v0.0.0" is what we want.